Menu/Ace_Menu_Engine.rb

```diff
@@ -324,16 +325,16 @@ def draw_actor_simple_status(actor, dx, dy)
     draw_actor_icons(actor, dx, dy + line_height * 2)
     dw = contents.width - dx - 124
     draw_actor_class(actor, dx + 120, dy, dw)
-    draw_actor_hp(actor, dx + 120, dy + line_height * 2, dw)
+    draw_actor_hp(actor, dx + 120, dy + line_height * 1, dw)
     if YEA::MENU::DRAW_TP_GAUGE && actor.draw_tp? && !actor.draw_mp?
-      draw_actor_tp(actor, dx + 120, dy + line_height * 1, dw)
+      draw_actor_tp(actor, dx + 120, dy + line_height * 2, dw)
```

Comment:
Changed the position of the TP bar when there are no skills that use MP but they do use TP.